### PR TITLE
Release/0.1.0.beta1

### PR DESCRIPTION
**Pull request by engarpe**
### New features
- Create and deploy models locally and remotely with Amazon SageMaker
- Version your AI models
    - Create copies of models whenever you adjust weights or other parameters
    - Every model version remains operational and accessible
- Adjust weights and other parameters on your models
- Add models to a project
- Assign models for pre-labeling
    - Use a model to provide suggestions for human labelers to accept, adjust, or ignore in the labeling interface
- Directly predict using deployed models
    - Use prediction to debug your model or provide a direct labeled output
- Deploy and test models locally with new CLI tools

### Enhancements
- Enjoy more detailed SDK log outputs thanks to [Rich library](https://github.com/willmcgugan/rich) implementation
- Automatically verify model responses to ensure compatibility with super.AI schema
- Shorter build time for AI containers through staged builds using a source-to-image package

### User-facing change?
- [ ] No. Skip the rest of this section
- [x] Yes. Give a description of this change to be included in the release notes.

### What component(s) does this PR affect?

**Components**
- [ ] CLI
- [ ] APIs
- [x] DataProgram SDK

**How should the PR be classified in the release notes? Choose one:**
- [ ] `breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `bug`-fix - A user-facing bug fix worth mentioning in the release notes
- [ ] `documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by ankitmishra262 — CHANGES_REQUESTED** [review]
Minor changes, I will push the fixes.

File: superai/meta_ai/dockerizer/sagemaker_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        arn_role="arn:aws:iam::185169359328:role/service-role/AmazonSageMaker-ExecutionRole-20180117T160866",
+        image_name="genre_model",
+        version="2",
+        model_url="s3://canotic-ai/meta_ai_models/saved_models/fairseq_entity_disambiguation_aidayago.tar.gz",
+    )
+    invoke_sagemaker_endpoint(
+        endpoint="DEMO-genre-model-2",
+        mime="application/json",
+        payload='{"data":{"sentences": ["Einstein was a [START_ENT] German [END_ENT] physicist."]}}',
+    )
```
[the remark this answers is not in the document]
Comment (ankitmishra262) [reply superai/meta_ai/dockerizer/sagemaker_endpoint.py]: Removed

File: superai/meta_ai/dockerizer/sagemaker_endpoint.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                      for :param mode="MultiModel" this is a folder location and
+                      for :param mode="SingleModel" this should be a location to a `tar.gz` file
+    :param initial_instance_count: Number of instances to serve on the endpoint
+    :param instance_type: Type of AWS instance to be used in the endpoint. Note that you cannot use GPU instances for
+                          :param mode="MultiModel"
+    :param mode: "SingleModel" or "MultiModel" type of instance to use
+    """
+    sm_client = boto3.client(service_name="sagemaker")
+    if arn_role is not None:
+        # arn_role = "arn:aws:iam::185169359328:role/service-role/AmazonSageMaker-ExecutionRole-20180117T160866"
```
[the remark this answers is not in the document]
Comment (ankitmishra262) [reply superai/meta_ai/dockerizer/sagemaker_endpoint.py]: removed

File: superai/meta_ai/dockerizer/sagemaker_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+        res = requests.post(url, data=payload, headers=headers)
+    if res.status_code == 200:
+        log.info(res.json())
+    else:
+        message = "Error , received error code {}: {}".format(res.status_code, res.text)
+        log.error(message)
+
+
+def get_sagemaker_runtime_client(target_model=None, mode="SingleModel", arn_role=None):
+    # arn_role = "arn:aws:iam::185169359328:role/service-role/AmazonSageMaker-ExecutionRole-20180117T160866"
```
[the remark this answers is not in the document]
Comment (ankitmishra262) [reply superai/meta_ai/dockerizer/sagemaker_endpoint.py]: removed

**Review by ankitmishra262 — APPROVED** [review]

**Review by lpfann — APPROVED** [review]

**Comment by ankitmishra262** [comment]
@engarpe another point to add on to, which you don't want to handle during your break, I am not authorized to merge the PR. Would it be good to give that permission to a few of us?

**Pull request merged**